components/browser/errorpages/src/main/assets/parseQueryAndInject.js

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Handles the parsing of the ErrorPages URI and then passes them to injectValues
+ */
+window.parseQuery = function(queryString) {
+   var query = {};
+   var pairs = (queryString[0] === '?' ? queryString.substr(1) : queryString).split('&');
```

Comment:
Could [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) be used here to simplify the parsing code?